value-scenarios/README.md

```diff
@@ -0,0 +1,22 @@
+# Value Scenarios
+This directory is scenarios showing scenarios
```

Comment:
I got lost at `sceanrios showing scenarios`. Maybe we can change the sentence?

Like:
> This directory provides scenarios that demonstrate the value of some aspects of EoX.

Reply:
done